src/common/daft-config/src/lib.rs

```diff
@@ -75,6 +76,7 @@ impl Default for DaftExecutionConfig {
             csv_target_filesize: 512 * 1024 * 1024, // 512MB
             csv_inflation_factor: 0.5,
             shuffle_aggregation_default_partitions: 200,
+            shuffle_join_default_partitions: 200,
```

Comment:
This should now be 16, see: https://github.com/Eventual-Inc/Daft/pull/2844#discussion_r1765438345